resources/wren/shaders/pbr.frag

```diff
@@ -209,9 +210,8 @@ vec3 PBRpass(vec3 l, vec3 n, vec3 v, vec3 h, vec4 lightColorAndIntensity, float
 }
 
 void main() {
-  vec3 viewFragmentNormal = normalize(fragmentNormal);
-
-  fragNormal = vec4(normalize(viewFragmentNormal), 1.0) * 0.5 + 0.5;
+  vec3 viewFragmentNormal = (reverseNormals > 0) ? normalize(-fragmentNormal) : normalize(fragmentNormal);
+  fragNormal = vec4(normalize(viewFragmentNormal), 1.0) * 1.0;
```

Comment:
This seems odd, why should we multiply by 1.0? It seems useless. Are you sure this change is correct?

Reply:
This was a simply test that should not have been committed, thank you for spotting this!